content/docs/guides/iframes@es.md

```diff
@@ -1,60 +1,63 @@
 ---
-$title: Incluir Iframes
+$title: Incluir iframes
+$order: 2
+$category: Develop 
+components:
+  - iframe
+toc: true
 ---
 [TOC]
 
-Aprenda cómo mostrar contenido multimedia en sus páginas y cómo utilizar iframes
-para mostrar contenido avanzado fuera de las limitaciones de AMP.
+Descubre cómo incluir contenido multimedia en tus páginas y cómo usar iframes para mostrar contenido avanzado saltándose las limitaciones de AMP.
 
-## Lo básico
+## Aspectos básicos
 
-Muestra un iframe en tu página usando el elemento 
+Incluye un iframe en tu página usando el elemento 
 [`amp-iframe`](/es/docs/reference/components/amp-iframe.html).
 
-Los iframes son especialmente útiles en AMP para mostrar contenido no
-soportado en el contexto de la página principal, como el contenido que requiere JavaScript.
+Los iframes son especialmente útiles en AMP para mostrar contenido que no se admita en 
+la página principal, como el contenido que requiere JavaScript creado por usuarios.
 
-### Requerimientos del elemento `amp-iframe`
+### Requisitos de `amp-iframe`:
 
-* Debe ser al menos de **600px** o **75%** de la primera ventana de visualización lejos de la parte superior.
-* Solo puede solicitar recursos a través de HTTPS, y no deben estar en el mismo origen que el contenedor, a menos que no especifique allow-same-origin.
+* Debe tener al menos **600 píxeles** o un **75%** de la primera ventana gráfica si empezamos por el principio de la página.
+* Solo puede solicitar recursos por HTTPS, y estos no deben tener el mismo origen que el contenedor salvo que no se especifique allow-same-origin.
 
-{% call callout('Read on', type='read') %}
-Aprende más sobre [lista de especificación para <code>amp-iframe</code>](/es/docs/reference/components/amp-iframe.html).
+{% call callout('Consejo', type='read') %}
```

Comment:
"Read on" is "Leer más" 
If change this, all notes with "read on" must be changed to "Consejo" :-1: